### DISCUSSION TITLE
developer/server/home

### DISCUSSION THREAD

**Discussion by giscus** [devlive-community/datacap#808] [Q&A] [answered]
# developer/server/home

DataCap is integrated software for data transformation, integration and visualization.

https://datacap.devlive.org/developer/server/home.html?h=parser

**Comment by tsstnt** [comment]
运行server 提示java: 程序包io.edurt.datacap.parser.antlr不存在

**Reply by qianmoQ** [reply]
需要先执行 `./mvnw -T 1C clean install package -Dspotbugs.skip -Dgpg.skip -Dcheckstyle.skip -DskipTests=true`

**Reply by baiyekaslana** [reply]
使用window中的IDEA在终端输入.\mvnw.cmd后边的参数要加啥
[ERROR] Unknown lifecycle phase ".skip". 
一直报这个错误，无法识别 .skip 作为一个有效的生命周期阶段或目标

**Reply by qianmoQ** [reply]
目前暂未适配 windows 命令行

**Reply by baiyekaslana** [reply]
那windows中的idea可以打包嘛，我这里一直显示有五个依赖项无法解析

**Reply by baiyekaslana** [reply]
未解析的依赖项: 'io.edurt.datacap:datacap-shaded-pinot:jar:2024.3.10-SNAPSHOT'
未解析的依赖项: 'io.edurt.datacap:datacap-shaded-ydb:jar:2024.3.10-SNAPSHOT'

**Reply by baiyekaslana** [reply]
windows和mac的区别应该不会影响吧

**Reply by baiyekaslana** [reply]
现在使用jdk1.8，在后面打包的时候就会提示使用jdk11，使用jdk11的话，就会提示没有tool.jar,这个问题要如何解决

**Reply by qianmoQ** [reply]
需要整体编译一下

**Reply by baiyekaslana** [reply]
整体编译不了

**Reply by qianmoQ** [reply]
./mvnw -T 1C clean install package -Dspotbugs.skip -Dgpg.skip -Dcheckstyle.skip -DskipTests=true

**Reply by baiyekaslana** [reply]
[ERROR] Unknown lifecycle phase ".skip".
一直报这个错误，无法识别 .skip 作为一个有效的生命周期阶段或目标

**Reply by baiyekaslana** [reply, marked as answer]
这个错误是windows中，终端需要调用cmd，而不是powerShell，现在这个命令要是在powerShell中运行的话，需要在参数两边加单引号引用.\mvnw -T 1C clean install package '-Dspotbugs.skip' '-Dgpg.skip' '-Dcheckstyle.skip' -DskipTests=true

**Comment by baiyekaslana** [comment]
现在使用jdk1.8，在后面打包的时候就会提示使用jdk11，使用jdk11的话，就会提示没有tool.jar,这个问题要如何解决

**Reply by baiyekaslana** [reply]
tools.jar

**Reply by qianmoQ** [reply]
这个jar已经放上去了

**Reply by baiyekaslana** [reply]
您这个项目用的jdk和maven用的啥版本

**Comment by jingming99** [comment]
请问服务端开发 sql 应该执行那个sql 文件？另外，微信群二维码失效，麻烦更新下

**Reply by jingming99** [reply]
版本是 2024.4.1

**Reply by qianmoQ** [reply]
https://github.com/devlive-community/datacap/blob/dev/configure/schema/datacap.sql

**Reply by qianmoQ** [reply]
二维码已更新
